umbraco/Umbraco.CMS.Backoffice
# Feature: Entity Bulk Action `trash` kind

## Description

Adds a reusable Entity Bulk Action `trash` kind, and implements this for the Media collection bulk trash feature.

## Types of changes


## How to test?

- Ensure that "Allow bulk trash" is enabled in the Collection (data-type) configuration.
- In the Media section, (at the root section view, or inside a Folder detail view), select 1 or more media items.
- In the Collections actions toolbar (at the bottom), select the "Trash" action button.
- The items will be moved to the Recycle Bin, a success notification should be displayed and the Media Collection UI should be automatically refreshed.
- Expand the Recycle Bin to check that the trashed media items are there.
